src/App.jsx

```diff
@@ -10,14 +9,23 @@ import Home from './routes/Home';
 import User from './routes/User';
 import Create from './routes/Create';
 import './App.css';
+import BlueBear from './assets/blue_bear.png';
 
 function App() {
+  const onAddEntry = async (communityId, selected) => {
+    console.log(selected);
+  };
   const { setUserData } = useCurrentUserData();
 
   useEffect(() => {
     const unsubscribe = onAuthStateChanged(auth, (user) => {
       if (user) {
-        setUserData({ username: user.displayName, email: user.email, userid: user.uid });
+        setUserData({
+          username: user.displayName,
+          email: user.email,
+          userid: user.uid,
+          photo: BlueBear
```

Comment:
if you want random profile, you can make an array of each color of bear image and get random number within the length of the array and assign it